verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -82,87 +96,112 @@ export default class NewCaseForm extends React.Component<
+                                    disabled={isSubmitting}
+                                    onClick={submitForm}
                                 >
-                                    <MenuItem value={undefined}></MenuItem>
-                                    <MenuItem value={'Female'}>Female</MenuItem>
-                                    <MenuItem value={'Male'}>Male</MenuItem>
-                                </Field>
-                            </FormControl>
-                        </fieldset>
-                        <fieldset>
-                            <legend>Location</legend>
-                            <Field
-                                label="Country"
-                                name="country"
-                                type="text"
-                                component={TextField}
-                            />
-                        </fieldset>
-                        <fieldset>
-                            <legend>Events</legend>
-                            <InputLabel htmlFor="confirmedDate">
-                                Date confirmed
-                            </InputLabel>
-                            <Field name="confirmedDate" type="date" />
-                        </fieldset>
-                        <fieldset>
-                            <legend>Source</legend>
-                            <Field
-                                label="Source URL"
-                                name="sourceUrl"
-                                type="text"
-                                placeholder="https://..."
-                                component={TextField}
-                            />
-                        </fieldset>
-                        <fieldset>
-                            <legend>Notes</legend>
-                            <Field
-                                label="Notes"
-                                name="notes"
-                                type="text"
-                                component={TextField}
-                            />
-                        </fieldset>
-                        {isSubmitting && <LinearProgress />}
-                        <br />
-                        <Button
-                            variant="contained"
-                            color="primary"
-                            data-testid="submit"
-                            disabled={isSubmitting}
-                            onClick={submitForm}
-                        >
-                            Submit case
-                        </Button>
-                        {this.state.errorMessage && (
-                            <h3> {this.state.errorMessage as string} </h3>
+                                    Submit case
+                                </Button>
+                                {this.state.errorMessage && (
+                                    <h3>
+                                        {' '}
```

Comment:
what is this? some kind of emoji?
\\{'o'}/

just kidding, but maybe just use &nbsp; here instead or some left margin style if you want the error message indented?

Reply:
Oh weird, good catch. The formatter added those in because in the old version there were spaces between the h3 tags and the {this....}. Removed them.